sdk/clientcore/core/src/main/java/io/clientcore/core/http/pipeline/HttpLoggingPolicy.java

```diff
@@ -11,68 +11,57 @@
  */
 public class HttpLoggingPolicy implements HttpPipelinePolicy {
+    private static final HttpLogOptions DEFAULT_HTTP_LOG_OPTIONS = new HttpLogOptions();
+    private static final Set<HttpHeaderName> ALWAYS_ALLOWED_HEADERS = Set.of(TRACEPARENT);
     private static final int MAX_BODY_LOG_SIZE = 1024 * 16;
     private static final String REDACTED_PLACEHOLDER = "REDACTED";
     private static final ClientLogger LOGGER = new ClientLogger(HttpLoggingPolicy.class);
     private final HttpLogOptions.HttpLogDetailLevel httpLogDetailLevel;
     private final Set<HttpHeaderName> allowedHeaderNames;
+
     private final Set<String> allowedQueryParameterNames;
-    private final HttpRequestLogger requestLogger;
-    private final HttpResponseLogger responseLogger;
 
-    private static final String REQUEST_LOG_MESSAGE = "HTTP request";
-    private static final String RESPONSE_LOG_MESSAGE = "HTTP response";
+    private static final String HTTP_REQUEST_EVENT_NAME = "http.request";
+    private static final String HTTP_RESPONSE_EVENT_NAME = "http.response";
+
+    // request log level is low (verbose) since almost all request details are also
+    // captured on the response log.
+    private static final ClientLogger.LogLevel HTTP_REQUEST_LOG_LEVEL = ClientLogger.LogLevel.VERBOSE;
+    private static final ClientLogger.LogLevel HTTP_RESPONSE_LOG_LEVEL = ClientLogger.LogLevel.INFORMATIONAL;
 
     /**
      * Creates an HttpLoggingPolicy with the given log configurations.
      *
      * @param httpLogOptions The HTTP logging configuration options.
      */
     public HttpLoggingPolicy(HttpLogOptions httpLogOptions) {
-        if (httpLogOptions == null) {
-            this.httpLogDetailLevel = HttpLogOptions.HttpLogDetailLevel.NONE;
-            this.allowedHeaderNames = Collections.emptySet();
-            this.allowedQueryParameterNames = Collections.emptySet();
-            this.requestLogger = new DefaultHttpRequestLogger();
-            this.responseLogger = new DefaultHttpResponseLogger();
-        } else {
-            this.httpLogDetailLevel = httpLogOptions.getLogLevel();
-            this.allowedHeaderNames = httpLogOptions.getAllowedHeaderNames();
-            this.allowedQueryParameterNames = httpLogOptions.getAllowedQueryParamNames()
-                .stream()
-                .map(queryParamName -> queryParamName.toLowerCase(Locale.ROOT))
-                .collect(Collectors.toSet());
-            this.requestLogger = (httpLogOptions.getRequestLogger() == null)
-                ? new DefaultHttpRequestLogger()
-                : httpLogOptions.getRequestLogger();
-            this.responseLogger = (httpLogOptions.getResponseLogger() == null)
-                ? new DefaultHttpResponseLogger()
-                : httpLogOptions.getResponseLogger();
-        }
+        HttpLogOptions logOptionsToUse = httpLogOptions == null ? DEFAULT_HTTP_LOG_OPTIONS : httpLogOptions;
```

Comment:
Doesn't this change behavior as `new HttpLogOptions()` will configure allowed header and query parameter names and use the default environment log level. Meaning disabling `HttpLoggingPolicy` will require:

```java
HttpLogOptions disabled = new HttpLogOptions()
    .setLogLevel(HttpLogDetailLevel.NONE)
    .setAllowedHeaderNames(Collections.emptyList())
    .setAllowedQueryParameterNames(Collections.emptyList());
```

Then have that passed where `HttpLoggingPolicy` should be disabled.